docs/source/task_summary.rst

```diff
@@ -108,6 +107,7 @@ each other. The process is the following:
     >>> sequence_1 = "Apples are especially bad for your health"
     >>> sequence_2 = "HuggingFace's headquarters are situated in Manhattan"
 
+    >>> # The tokekenizer will automatically add any model specific separators (i.e. <CLS> and <SEP>) and tokens to the sequence, as well as compute the attention masks.
```

Comment:
The original doc seemed like you wanted to emphasize that the tokenizer creates everything you need (masks, seperators, etc), so added a comment here so the reader can more easily follow this.